### DISCUSSION TITLE
Burn a profile into the image data

### DISCUSSION THREAD

**Discussion by fmw42** [ImageMagick/ImageMagick#4374] [Digital Image Processing]
Is there some way to apply a profile to an image so that the image pixels are actually changed and the profile can be removed and the image still looks as if it had the profile?

**Comment by snibgo** [comment]
When an image has no ICC colour profile, viewing software will assume that the image is encoded as sRGB.

If an image has an embedded sRGB profile, then removing the profile will cause almost no change to the appearance. (There may be a slight change because not all sRGB profiles are the same.)

But if an image has an embedded profile that is _not_ sRGB, then removing the profile will change the appearance.

So, to do what you want, take the image with embedded profile, and convert that with "-profile sRGB.icc" or some other sRGB profile to change the pixels, then remove the profile.

**Reply by fmw42** [reply]
Thanks snibgo. That was what I needed.
